pkg/ytconfig/node.go

```diff
@@ -386,6 +421,16 @@ func getExecNodeServerCarcass(spec *ytv1.ExecNodesSpec, usePorto bool) (ExecNode
 	c.JobResourceManager.ResourceLimits = c.ExecNode.JobController.ResourceLimitsLegacy
 	c.ExecNode.GpuManager = c.ExecNode.JobController.GpuManager
 
+	if usePorto {
+		for _, loc := range spec.Locations {
+			c.ExecNode.RootFSBinds = append(c.ExecNode.RootFSBinds, RootFSBind{
+				InternalPath: loc.Path,
+				ExternalPath: loc.Path,
+				ReadOnly:     false,
+			})
```

Comment:
Could you please add some comment with explaining why rootfs binds configuration is being built that way.